invisible_cities/cities/base_cities.py

```diff
@@ -422,17 +414,20 @@ def __init__(self, **kwds):
         self.rebin          = conf.rebin
         self.reco_algorithm = find_algorithm(conf.reco_algorithm)
 
-    def rebin_s2(self, S2, Si):
-        if self.rebin <= 1:
-            return S2, Si
-
-        S2_rebin = {}
-        Si_rebin = {}
-        for peak in S2:
-            t, e, sipms = cpf.rebin_S2(S2[peak][0], S2[peak][1], Si[peak], self.rebin)
-            S2_rebin[peak] = Peak(t, e)
-            Si_rebin[peak] = sipms
-        return S2_rebin, Si_rebin
+    #TODO ALEX SHOULD REVISE
+
+    # def rebin_s2(self, s2d, sid):
+    #     """rebins s2d and sid dictionaries"""
+    #     if self.rebin <= 1:
+    #         return s2d, sid
+    #
+    #     S2_rebin = {}
+    #     Si_rebin = {}
+    #     for peak in S2:
+    #         t, e, sipms = cpf.rebin_S2(S2[peak][0], S2[peak][1], Si[peak], self.rebin)
+    #         S2_rebin[peak] = Peak(t, e)
+    #         Si_rebin[peak] = sipms
+    #     return S2_rebin, Si_rebin
```

Comment:
Yes, but there is a point in having the possibility of re-rebinning S2 and S2Si at Dorothea level. Typically you have rebinned the s2 and s2si in bins of 1 mus when creating the pmaps. This bin is often "too thin" (it corresponds to 1 mm) and when creating hits and tracks we will probably increase to 5 mus or even 10 mus. Thus we must be able to rebin by a factor 5 or even 10 at this level. The idea is to call from here @abotas new rebinning functions. Ale, can you do this?